### PR TITLE
triangular_treasure

### PR DESCRIPTION
**Pull request by Alexander-Andrade**
Description:

Triangular numbers are so called because of the equilateral triangular shape that they occupy when laid out as dots. i.e.

1st (1)   2nd (3)    3rd (6)

---

```
       *         **
                 *
```

You need to return the nth triangular number. You should return 0 for out of range values:

  triangular(0)==0,
  triangular(2)==3,
  triangular(3)==6,
  triangular(-10)==0

### CODE REVIEW

File: asana_tasks/triangular_treasure/triangular_treasure.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #   triangular(3)==6,
 #   triangular(-10)==0
 
+
 def triangular( n )
   return 0 if n <= 0
   return n*n - triangular(n - 1)
```
Comment (valadzko) [inline asana_tasks/triangular_treasure/triangular_treasure.rb:17]: It is not necessary, but you can use ternary operator here, too.

```
def triangular(n)
  n <= 0 ? 0 : n*n - triangular(n - 1)
end
```

Comment (valadzko) [inline asana_tasks/triangular_treasure/triangular_treasure.rb:17]: @Alexander-Andrade Just for the education purposes, please change the function to have ternary operator and make one more commit to this branch. Let me know when it is done, so I will check pull request again.

**Comment by Alexander-Andrade** [comment]
done

**Comment by valadzko** [comment]
Looks ok! Now you can close pull request and remove remote and local branches.

**Pull request closed without merging**